src/styles/elements/_product-card.scss

```diff
@@ -0,0 +1,112 @@
+.product-card {
+  width: $card-width;
+  background-color: $card-bg-color;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  transition: all 0.3s ease 0s;
+
+  &__container {
+    width: 100%;
+    padding-top: 32px;
+    padding-inline: 16px;
+    padding-bottom: 16px;
+  }
+
+  &__image {
+    position: relative;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px;
+    margin-bottom: 40px;
+    img {
```

Comment:
do not use a combined selector without the necessary
 Also, do not use tag selector
 
 Instead add class to the `img` tag and use it to add styles